.github/workflows/pr-validation-fork.yml

```
@@ -94,7 +94,8 @@ jobs:
       - name: Run integration tests
         run: |
           container_id=${{steps.devcontainer.outputs.id}}
-          docker exec -e AZURE_TENANT_ID -e AZURE_CLIENT_ID -e AZURE_CLIENT_SECRET -e AZURE_SUBSCRIPTION_ID -e AZURE_CLIENT_SECRET_MULTITENANT -e AZURE_CLIENT_ID_MULTITENANT "$container_id" task controller:ci-integration-tests --concurrency 2 # limit concurrency because build machines are small
+          docker exec -e AZURE_TENANT_ID -e AZURE_CLIENT_ID -e AZURE_CLIENT_SECRET -e AZURE_SUBSCRIPTION_ID -e AZURE_CLIENT_SECRET_MULTITENANT -e AZURE_CLIENT_ID_MULTITENANT "$container_id" task controller:ci-integration-tests --concurrency 3 # limit concurrency because build machines are small
+          # Change concurrency to --concurrency 1 once https://github.com/go-task/task/pull/1025 is merged
```

```suggestion
          # Change concurrency to --concurrency 2 once https://github.com/go-task/task/pull/1025 is merged
```

Unless we wanted 1 before but had deadlock issues?

I thought we picked 2 because the build machines have 2 cores, in which case a default of 2 seems OK to me?

---

Same comment applies below as well.